src/main/scala/glue/data/Writer.scala

```diff
@@ -3,6 +3,17 @@ package data
 
 import glue.typeclass.{Applicative, Functor, Monad, Monoid}
 
+/*
+TODO: Replace this implementation with:
+
+  type Writer[W, A] = WriterT[Id, W, A]
+  object Writer {
+    def apply[W, A](w: W, a: A): Writer[W, A] = WriterT[Id, W, A]((w, a))
+    def tell[W](w: W): Writer[W, Unit] = WriterT.tell(w)
+    def value[W: Monoid, A](a: A): Writer[W, A] = WriterT.value(a)
+  }
+*/
+
```

Comment:
We could remove this file and put the alias and the companion object definition in this commented code where the other aliases are defined in `package.scala`.